server-side-rendering/website1/build/webpack.config.js/server.base.js

```diff
@@ -35,10 +40,9 @@ module.exports = merge.smart(common, {
       library: { type: "commonjs2" },
       filename: "container.js",
       remotes: {
-        website2: path.resolve(
-          __dirname,
-          "../../../website2/buildServer/container.js"
-        ),
+        website2: {
```

Comment:
this isn't important, but useful to see when it requires the remote during the runtime